src/collector/glide.yaml

```diff
@@ -0,0 +1,76 @@
+package: github.com/m3db/m3collector
```

Comment:
can you migrate deps to central glide and delete this and glide.lock in the subfolder